### DISCUSSION TITLE
How to change a color of `gum format`?

### DISCUSSION THREAD

**Discussion by 5ouma** [charmbracelet/gum#281] [Q&A] [answered]
As for `gum input`, `gum spin` or other commands can be changed its colors.
But for `gum format`, there're no options to change colors.
Is there any method to do this? Please tell me the way.

**Comment by MikaelFangel** [comment, marked as answer]
You can change themes for `gum format` using the `--theme` option. Some supported themes are:
- dark
- dracula
- notty

If you want a custom theme, you can pass in a glamour JSON theme-file. - Source: [https://github.com/charmbracelet/gum/releases/tag/v0.9.0](https://github.com/charmbracelet/gum/releases/tag/v0.9.0)

You can create theme files using this guide: [https://github.com/charmbracelet/glamour/tree/master/styles](https://github.com/charmbracelet/glamour/tree/master/styles)

**Reply by 5ouma** [reply]
Thanks for your help!!
